apps/org.videolan.VLC/eos-vlc-helper.appdata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>vlc.desktop</id>
```

Comment:
Should that be "eos-vlc.desktop" ? And that file I would actually name "eos-vlc.appdata.xml", so you can get rid of the "helper" part.